assignments/callbacks.js

```diff
@@ -24,91 +24,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum','yo-yo', 'Gum'];
 
 */
 
-
- 
-
 function getLength(arr, cb) {
-  return cb();
+  return cb(arr.length);
 }
-console.log(getLength(items, function () {
-  return items.length;
-}) );
- 
-  // getLength passes the length of the array into the callback.
+getLength(items, function(length) {
+  console.log(length);
+});
 
+// getLength passes the length of the array into the callback.
 
 function last(arr, cb) {
-  return cb();
+  return cb(arr.last);
```

Comment:
Here you're using the dot notation on an array.  On arrays, dot notation is to access methods/properties but arrays don't have a `last` property.

items.slice(-1)[0] will return you the last item of the array, but you're doing that on the function invocation and not inside the function declaration.